api/pom.xml

```diff
@@ -31,6 +31,13 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
+            <scope>provided</scope>
```

Comment:
As `provided` and the exclusions are defined in the root `dependencyManagement` section, does it need to be duplicated here?